03_capabilities.html

```diff
@@ -1 +1,179 @@
-<!-- Section removed as it was describing specific local end implementations -->
\ No newline at end of file
+<section>
+  <h2>Capabilities</h2>
+  <p>
+    Webdriver <dfn>capabilities</dfn> allow the <a>local end</a> to specify what
+     features it requires the <a>remote end</a> to fulfil to be able to create a
+      <a>new session</a>
+  </p>
+  <h3>Processing Capabilities</h3>
+  <p>
+    <p>The <a>remote end steps</a> to <dfn>process capabilities</dfn> are:</p>
+    <ol>
+      <li><p>Let <var>server capabilities</var> be a JSON Object with the following entries:<p></li>
+        <ol>
+          <li><p>Let "browserName" be an entry with value equal to the lowercase name of the user agent</p></li>
+          <li><p>Let "browserVersion" be an entry with the value equal to the version of the user agent<p></li>
+          <li><p>Let "platformName" be an entry with the value equal to the lowercase name of the platform.</p></li>
+          <li><p>Let "platformVersion" be an entry with the value equal to the version of the platform</p></li>
+          <li><p>Let "acceptSslCerts" be an entry with the value equal to true if it can handle
+            <a href='#invalid-ssl-certificates'>Invalid SSL Certifications</a> else let it be equal to
+            false.</p></li>
+          <li><p>Let "takesScreenshot" be an entry with the value equal to true if it can capture
+          a screenshot of the viewport described in <a>Take Screenshot</a>.</p></li>
```

Comment:
reading http://www.w3.org/2014/10/30-testing-minutes.html#item11 we do still support them